integrations/sources/webhook.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Ingest data from Webhook"
```

Comment:
Let's use "webhooks" here. It's a general technology / mechanism that do not need to be captalized.